docs/UserManagementService

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Is this the API collection for this service? If so, what is the file type(extension file name)? Re-upload the proper & updated file with correct extension(.yml/json).

Reply:
done